circle.yml

```diff
@@ -30,11 +37,16 @@ deployment:
   release:
     branch: master
     commands:
-      - 'cd $IMPORT_PATH && chmod +x ./build.sh && ./build.sh'
+      - 'cd $IMPORT_PATH && chmod +x ./build.sh && ./build.sh'      
+      - source .env; docker build -t quay.io/turner/harbor-compose:${VERSION} $IMPORT_PATH
       - 'cd $IMPORT_PATH && ghr -t $GITHUB_TOKEN -u $CIRCLE_PROJECT_USERNAME -r $CIRCLE_PROJECT_REPONAME --replace `git describe --tags` dist/'
+      - source .env; docker push quay.io/turner/harbor-compose:${VERSION}
+      - docker push quay.io/turner/harbor-compose:latest
```

Comment:
well it's hard to merge to master and test at the same time. Only reason I know, is because I did it. :) don't tell anyone.